coalib/core/DependencyTracker.py

```diff
@@ -293,7 +293,7 @@ def check_circular_dependencies(self):
          ...
         coalib.core.CircularDependencyError.CircularDependencyError: ...
 
-        :raises coalib.core.CircularDependencyError.CircularDependencyError:
+        :raises CircularDependencyError:
```

Comment:
@userzimmermann, yes the linkg to the exception class is correctly generated if we use the short form